dags/main_summary.py

```diff
@@ -566,18 +567,42 @@
     output_visibility="private",
     dag=dag)
 
-taar_collaborative_recommender = EMRSparkOperator(
+taar_collaborative_recommender = SubDagOperator(
     task_id="addon_recommender",
-    job_name="Train the Collaborative Addon Recommender",
-    execution_timeout=timedelta(hours=10),
-    instance_count=20,
-    owner="mlopatka@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "mlopatka@mozilla.com", "vng@mozilla.com"],
-    env={"date": "{{ ds_nodash }}",
-         "privateBucket": "{{ task.__class__.private_output_bucket }}",
-         "publicBucket": "{{ task.__class__.public_output_bucket }}"},
-    uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/addon_recommender.sh",
-    dag=dag)
+    subdag=moz_dataproc_jar_runner(
+        parent_dag_name=dag.dag_id,
+        dag_name="addon_recommender",
+        job_name="Train the Collaborative Addon Recommender",
```

Comment:
This will fail because of whitespaces (we're not sanitizing job names in current operator implementation).

Reply:
```suggestion
        job_name="Train_the_Collaborative_Addon_Recommender",
```